ocaml/xapi/xapi_globs.ml

```diff
@@ -1023,6 +1023,8 @@ let python3_path = ref "/usr/bin/python3"
 let observer_experimental_components =
   ref (StringSet.singleton Constants.observer_component_smapi)
```

Comment:
I would suggest to collect this functionality in a local module:
```
module PoolRec = struct
 ...
end
```

Reply:
You mean put `let recommendations t = ...` into a local module?